src/components/AssignTeamPopover/tests/__snapshots__/index.test.tsx.snap

```diff
@@ -0,0 +1,184 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
Do you think that this snapshot can be made smaller, in the spirit of https://kentcdodds.com/blog/effective-snapshot-testing ?

Reply:
> Do you think that this snapshot can be made smaller, in the spirit of https://kentcdodds.com/blog/effective-snapshot-testing ?

Yes. After reading some articles I learnt that one should make the assumption that the children components were tested and the parent should only care if they exist

Reply:
I will be pushing an update and tell me what you think